mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/scheduler/MantisSchedulerFactoryImpl.java

```diff
@@ -39,13 +37,12 @@ public class MantisSchedulerFactoryImpl implements MantisSchedulerFactory {
     private final ResourceClusters resourceClusters;
     private final ExecuteStageRequestFactory executeStageRequestFactory;
     private final JobMessageRouter jobMessageRouter;
-    private final SchedulingService mesosSchedulingService;
     private final MasterConfiguration masterConfiguration;
     private final MetricsRegistry metricsRegistry;
     private final Map<ClusterID, MantisScheduler> actorRefMap = new ConcurrentHashMap<>();
 
     @Override
-    public MantisScheduler forClusterID(@Nullable ClusterID clusterID) {
+    public MantisScheduler forClusterID(ClusterID clusterID) {
         Optional<ClusterID> clusterIDOptional = Optional.ofNullable(clusterID);
```

Comment:
If a field is not marked @nullable, it should assumed to be not null. We can get rid of this check here.